File: samples/python/basic_push/webhook_push.py
# tested on python 3.9

import urllib, json
import urllib.request

# change those lines to fit to your device (check it on https://my.smiirl.com) and your data
counterMac = 'e08e3c37e386'
counterToken = 'cb8822a2b6ef2b1391aa1ba55c08c5b5'
numberToShow = 12345

url = "http://api.smiirl.com/" + counterMac + "/set-number/" + counterToken + "/" + str(numberToShow)
req = urllib.request.Request(url)
r = urllib.request.urlopen(req).read()
cont = json.loads(r.decode('utf-8'))
found = False

# parsing json
for item in cont:
    if item == 'status' and cont[item] == 200:
        # print("number successfully pushed")
        found = True
# if(not found):
#    print("number push failed")

File: samples/php/basic_push/webhook_push.php
<?php

require_once("../../../src/php/SmiirlLibrary.php");
$smiirlLib = new SmiirlLibraryPhp();

// get the real parameters of your counter on https://my.smiirl.com
$counterCurlUrl = "http://api.smiirl.com/YOUR_COUNTER_MAC/set-number/YOUR_COUNTER_TOKEN/54321";

list($counterMac, $counterToken) = $smiirlLib->listCurlUrlAccessParameters($counterCurlUrl);
$smiirlLib->pushNumberOnCounter($counterMac, $counterToken, "4242");

